### DISCUSSION TITLE
How to Run Openfast on Linux System?

### DISCUSSION THREAD

**Discussion by icnosvbovwb** [OpenFAST/openfast#1667] [Q&A]
Dear @jjonkman 
I want to run Openfast on a LINUX operating system platform, so how do I compile it into a Linux recognized call library function instead of a DLL, such as OpenFAST Simulink_ Win32. so. What software do I need to install on Windows 10 Education 64-bit? Can you specify it specifically?

**Comment by deslaughter** [comment]
@icnosvbovwb , to run OpenFAST on a Linux system, it will need to be compiled for that system. It is not currently possible to compile a Linux executable on Windows. The documentation provides instructions for [compiling from source on Linux](https://openfast.readthedocs.io/en/dev/source/install/index.html#compile-from-source).
